kubernetes/structure_persistent_volume_spec.go

```diff
@@ -1028,6 +1034,14 @@ func patchPersistentVolumeSpec(pathPrefix, prefix string, d *schema.ResourceData
 			})
 		}
 	}
+	if d.HasChange(prefix + "node_affinity") {
+		v := d.Get(prefix + "node_affinity").([]interface{})
+		nodeAffinity := expandVolumeNodeAffinity(v)
+		ops = append(ops, &ReplaceOperation{
```

Comment:
I'm not entirely sure it's valid to pass complex types like `VolumeNodeAffinity` directly into a patch operation.

Would you mind adding a new acceptance test that exercises this patch operation?

Reply:
I see the RBDPersistentVolumeSource is also a similar complex type, so I think that is valid to pass.

Updated the acceptance test to cover more operations.